shared/src/commonMain/kotlin/co/nimblehq/blisskmmic/presentation/model/SurveySubmissionUiModel.kt

```diff
@@ -0,0 +1,28 @@
+package co.nimblehq.blisskmmic.presentation.model
+
+import co.nimblehq.blisskmmic.domain.model.SurveySubmission
+
+data class SurveySubmissionUiModel(
```

Comment:
shouldnt this be `SurveySubmissionUIModel` since UI is an abbreviation, both letters should be capitalised